src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/MsiInstallerStrings_zh_CN.wxl

```diff
@@ -1,5 +1,5 @@
 <?xml version = '1.0' encoding = 'utf-8'?>
-<WixLocalization Culture="zh-cn" xmlns="http://schemas.microsoft.com/wix/2006/localization" Codepage="936">
+<WixLocalization Culture="zh-CN" xmlns="http://schemas.microsoft.com/wix/2006/localization" Codepage="936">

```

Comment:
Please revert the value of the Culture attribute.

Reply:
Resolved in recent commit